doc/classes/Geometry2D.xml

```diff
@@ -201,12 +201,20 @@
 				Each polygon's vertices will be rounded as determined by [code]join_type[/code], see [enum PolyJoinType].
 				The operation may result in an outer polygon (boundary) and inner polygon (hole) produced which could be distinguished by calling [method is_polygon_clockwise].
 				[b]Note:[/b] To translate the polygon's vertices specifically, multiply them to a [Transform2D]:
-				[codeblock]
+				[codeblocks]
+				[gdscript]
 				var polygon = PackedVector2Array([Vector2(0, 0), Vector2(100, 0), Vector2(100, 100), Vector2(0, 100)])
 				var offset = Vector2(50, 50)
 				polygon = Transform2D(0, offset) * polygon
 				print(polygon) # prints [Vector2(50, 50), Vector2(150, 50), Vector2(150, 150), Vector2(50, 150)]
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				var polygon = new Vector2[] { new Vector2(0, 0), new Vector2(100, 0), new Vector2(100, 100), new Vector2(0, 100) };
+				var offset = new Vector2(50, 50);
+				polygon = (Vector2[]) new Transform2D(0, offset).Xform(polygon);
+				GD.Print(polygon); // prints [Vector2(50, 50), Vector2(150, 50), Vector2(150, 150), Vector2(50, 150)]
```

Comment:
`.Xform` with anything other than a `Vector2` is not currently valid in C#'s Transform2D. This can be changed, but right now `xform` doesn't exist in GDScript in master either, so I don't want to touch it until it's decided how it will function. Also, the GDScript example code uses the `*` operator, so I'm not sure if we want to support that too. In any case we want to have feature consistency between GDScript and C# where possible.

I think we can just leave a TODO note here for later:

```suggestion
				// TODO: This code is not valid right now. Ping @aaronfranke about it before Godot 4.0 is out.
				//polygon = (Vector2[]) new Transform2D(0, offset).Xform(polygon);
				//GD.Print(polygon); // prints [Vector2(50, 50), Vector2(150, 50), Vector2(150, 150), Vector2(50, 150)]
```